### ISSUE TITLE
cosmos auth module does not exist

### ISSUE THREAD

**Issue by grikomsn** [hyperweb-io/interchain-js-query#7]
https://github.com/search?q=repo:cosmology-tech/interchain+cosmos/auth/&type=code

https://github.com/cosmology-tech/interchain/blob/a84d48c97e15b80306c298f30e1064b6d6b60775/types/codegen/ibc/applications/interchain_accounts/v1/account.d.ts#L1

https://github.com/cosmology-tech/interchain/blob/a84d48c97e15b80306c298f30e1064b6d6b60775/src/codegen/ibc/applications/interchain_accounts/v1/account.ts#L1

https://github.com/cosmology-tech/interchain/tree/main/src/codegen/cosmos/auth

**Comment by chalabi2** [comment]
@pyramation I have also run into this issue. Even in the package itself /auth/v1beta1/auth doesnt resolve to anything.

**Comment by MbBrainz** [comment]
Same for me! 

```shell
./node_modules/.pnpm/interchain@1.5.0/node_modules/interchain/module/codegen/ibc/applications/interchain_accounts/v1/account.js:1:0
Module not found: Can't resolve '../../../../cosmos/auth/v1beta1/auth'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./node_modules/.pnpm/interchain@1.5.0/node_modules/interchain/module/codegen/ibc/bundle.js
./node_modules/.pnpm/interchain@1.5.0/node_modules/interchain/module/codegen/index.js
./node_modules/.pnpm/interchain@1.5.0/node_modules/interchain/module/index.js
./components/StakingByChain/StakingByChain.tsx
```

**Comment by MbBrainz** [comment]
It might be caused by this line in  `./scripts/codegen.js` ( i added the arrow in the codeblock below)

```js
telescope({
  protoDirs,
  outPath,
  options: {
    removeUnusedImports: true, // testing...
    classesUseArrowFunctions: true,

    prototypes: {
      excluded: {
        packages: [
          'ibc.applications.fee.v1', // issue with parsing protos (LCD routes with nested objects in params)

          'cosmos.auth.v1beta1',  <------------ excluded package
          'cosmos.app.v1alpha1',
```

https://github.com/cosmology-tech/interchain/blob/ddadf31eee0df1222f3daf78f3788fe30bb5f288/scripts/codegen.js#L25C30-L25C30

**Comment by pyramation** [comment]
try now — been updated `interchain` published @ `1.7.0`

**Comment by MbBrainz** [comment]
Perfect! Issue Resolved ✅
